pkg/apis/componentconfig/install/install_test.go

```diff
@@ -37,17 +37,17 @@ func TestCodec(t *testing.T) {
 	if err := json.Unmarshal(data, &other); err != nil {
```

Comment:
odd to be unmarshalling directly into an internal object... I know it's not a change, but worth a `// FIXME`